cargo-auditable.schema.json

```diff
@@ -1,101 +1,123 @@
+          "type": "string"
+        },
+        {
+          "anyOf": [
+            {
+              "type": "string",
+              "const": "git"
+            },
+            {
+              "type": "object",
+              "required": [
+                "kind"
+              ],
+              "properties": {
                 "kind": {
-                    "description": "\"build\" or \"runtime\". May be omitted if set to \"runtime\". If it's both a build and a runtime dependency, \"runtime\" is recorded.",
-                    "allOf": [
-                        {
-                            "$ref": "#/definitions/DependencyKind"
-                        }
-                    ]
-                },
-                "name": {
-                    "description": "Crate name specified in the `name` field in Cargo.toml file. Examples: \"libc\", \"rand\"",
-                    "type": "string"
+                  "type": "string",
+                  "const": "git"
                 },
-                "root": {
-                    "description": "Whether this is the root package in the dependency tree. There should only be one root package. May be omitted if set to `false`.",
-                    "type": "boolean"
-                },
-                "source": {
-                    "description": "Currently \"git\", \"local\", \"crates.io\" or \"registry\". Designed to be extensible with other revision control systems, etc.",
-                    "allOf": [
-                        {
-                            "$ref": "#/definitions/Source"
-                        }
-                    ]
-                },
-                "version": {
-                    "description": "The package's version in the [semantic version](https://semver.org) format.",
-                    "type": "string"
+                "rev": {
+                  "description": "Commit hash pointing to specific revision",
+                  "type": [
+                    "string",
+                    "null"
+                  ]
                 }
+              }
             }
+          ]
         },
-        "Source": {
-            "description": "Serializes to \"git\", \"local\", \"crates.io\" or \"registry\". Designed to be extensible with other revision control systems, etc.",
-            "oneOf": [
-                {
-                    "type": "string",
-                    "enum": [
-                        "CratesIo",
-                        "Git",
-                        "Local",
-                        "Registry"
-                    ]
-                },
-                {
-                    "type": "object",
-                    "required": [
-                        "Other"
-                    ],
-                    "properties": {
-                        "Other": {
-                            "type": "string"
-                        }
-                    },
-                    "additionalProperties": false
-                }
-            ]
```

Comment:
It is not obvious at first glance but the new schema should be backwards compatible and as far as I am aware even more correct as the values for `enum` keyword used here are not exactly right (e.g. de-serialization expects `crates.io` rather than `CratesIo`).

Reply:
Yes, the schema should be re-generated.

The mismatch between the documentation and the generated schema is a bug, and will need to be fixed, but that doesn't have to be done in this PR.